Scenes/BeatControl/BeatControl.gd

```diff
@@ -32,7 +31,7 @@ func play_next_in_sequence():
 		KEYS.WAIT:
 			pass
 
-func _on_AudioStreamConductor_beat(total, in_measure):
+func _on_AudioStreamConductor_beat(total, _in_measure):
```

Comment:
Similar comment as Conductor. unused variable...though I could have sworn in a future update we ended up using the variable and that broke things in my build for awhile. May need to check this method on the `main` branch and make sure.

Reply:
uh yeah just checked and we're definitely using it now in main, so don't think we want this changes...and in my local build it's definitely back to being `in_measure` ... not sure what gives.

Reply:
sigh I was looking at an older commit. what a noob.